TODO/the-art-of-minimalism-in-mobile-app-ui-design.md

```diff
@@ -1,91 +1,92 @@
+[雅虎天气](https://itunes.apple.com/us/app/yahoo!-weather/id628677149?mt=8) 显示了一张当前位置的风景图片，如果需要查看天气的详细信息，你只需要向上滑动便会马上显示出来。与在原先的图片上面叠加多一张图片相比，这种方式在增加了详细信息的易于获取的情况下还保存了上一张图片作为模糊背景。而且交互方式极为自然，你可以很方便的就返回到上应用的一层。![](http://babich.biz/content/images/2016/07/1-v6JG6G4UCwdS7XbXp0pZXA.jpeg)
 
-Yahoo! weather app for iOS
+IOS 上的雅虎天气
 
-## One App, One Typeface
+## 一个应用中只使用一种字体
 
-Mixing several different fonts can make your app seem fragmented and sloppy. Reducing the number of fonts on a screen can reveal the power of typography. When designing an app think about how can you make the typography powerful by playing with _weight, style and size_, not different typefaces.
+在一个应用中使用多种字体会看起来很散乱和马虎。减少屏幕上字体的类型数量可以增强排版的效果。当你在设计应用的时候你可以通过更改字体的权重，样式，尺寸和大小来优化布局效果，而不是更换字体。
 ![](http://babich.biz/content/images/2016/07/1-PuctKONH65PUaGAgj_IDPQ.png)
 
-In general, use a single font throughout your app. Image credit: Apple
+通常来说，一个应用中只使用一种字体 Image credit: Apple
 
-When you choosing a typeface for your app keep in mind that a safe bet is to rely on the platform’s default font:
+当你在为APP选择字体的时候，选择平台的默认字体可能是最安全稳妥的选择：
 
-*   Apple uses the San Francisco family of typefaces to provide consistent reading experience across all platforms (the San Francisco font for iOS 9 is called SF-UI).
-*   [Roboto](https://www.google.com/fonts/specimen/Roboto) and Noto are the standard typefaces on Google Android and Chrome. ![](http://babich.biz/content/images/2016/07/1-h9TgxM4LubkjVDIhEG5U4g.png)
+*  苹果使用_San Francisco family_字体来提供全平台一致的阅读体验。（在 IOS 9 中简称为 SF-UI）
+*   _[Roboto](https://www.google.com/fonts/specimen/Roboto)_ 和 _Noto_ 分别是 _Android_ 和 _Chrome_ 的默认字体 ![](http://babich.biz/content/images/2016/07/1-h9TgxM4LubkjVDIhEG5U4g.png)
 
-    Reducing the number of fonts on a screen can reveal the power of typography. Image credit: Dribbble
+    减少屏幕上使用的字体类型能够获得较好的排版效果。素材来源: Dribbble
 
-## Data Spotlight
+## 数据的视觉焦点
 
-You should use _big font size and striking color_ to make a _certain data_ the center of focus. Using neutral colors for the general scheme and adding contrasting colors for calls to action, helps the user focus on the action we want them to take.
+你应该使用大号字体和醒目的颜色来让特定的数据成为视觉的焦点。使用中性的颜色（黑白灰）来展示普通的内容，而一些具备操作的部分则使用强对比的颜色来吸引用户注意，从而给给予用户正确的指导和操作。
 ![](http://babich.biz/content/images/2016/07/0-pgeX-afdEGctxooq.jpg)
 
 Bright accent color into an otherwise-neutral palette is one of the easiest color schemes to create. It’s also one of the most striking, visually. Image credit: Smashing Magazine
```

Comment:
> Bright accent color into an otherwise-neutral palette is one of the easiest color schemes to create. It’s also one of the most striking, visually. Image credit: Smashing Magazine

这段应该但没有被删除